### ISSUE TITLE
Obtained different reconstruction results on Volta-GPU and Ampere-GPU

### ISSUE THREAD

**Issue by yanjh97** [DRosemei/RoMe#25]
Hi, @DRosemei . I tried to reconstruct the same clip using RoMe on Volta-GPU and Ampere-GPU seperately, but obtained different results (as shown below).
|Volta-GPU|Ampere-GPU|
|-|-|
|![image](https://github.com/DRosemei/RoMe/assets/54617163/23943ba9-8448-4431-b99e-f9b3b3781298)|![image](https://github.com/DRosemei/RoMe/assets/54617163/ae2b916a-38a6-4d5f-95e3-c296042d824a)|

It's obvious that the result on the Ampere-GPU is abnormal, with black dots arranged regularly. Have you encountered this problem?

**Comment by DRosemei** [comment]
@yanjh97 Hi, I'm sorry I can't tell you why because I only use RTX-3090. It may be bugs of pytorch3D and you can try to use [flat mesh](https://github.com/DRosemei/RoMe/blob/main/utils/geometry.py#L6).

**Comment by yanjh97** [comment]
Hi, @DRosemei , I also tried on RTX 3090. The result is same as one on Ampere-GPU. I found that:
1. black vertices (that is, background label) occured only when x or y of vertices larger than a value. 
2. lines composed by these black vertices are paralleled with X or Y axis.
For example:
![image](https://github.com/DRosemei/RoMe/assets/54617163/19951a0f-7077-4e12-b503-d657ce2c148d)
![image](https://github.com/DRosemei/RoMe/assets/54617163/4663e572-bd47-4577-9802-c2fb91953793)
